src/test/resources/TestSuite.xml

```diff
@@ -2,7 +2,10 @@
 <suite name="Test Suite" verbose="1" configfailurepolicy="continue">
   <test name="Workshop tests">
     <classes>
-      <class name="ui.LoginTest"></class>
+      <class name="ui.LoginTest"/>
```

Comment:
we should create separate test suites for UI/API. e,g. ui-tests.xml and api-tests.xml